bin/storj-audit-tool.js

```diff
@@ -346,26 +348,34 @@ stream.on('data', function(line) {
                 }
 
                 res.on('data', (chunk) => {
+                  size += chunk.length;
                   hasher.update(chunk);
                   file.write(chunk);
                 });
 
                 res.on('end', () => {
                   const actual = storj.utils.rmd160b(hasher.digest()).toString('hex');
-                  if (actual == shard.hash) {
-                    shardResults[sanitizeNodeID(shard.hash)] = {
-                      status: SUCCESS,
-                      contract: contract.toObject()
-                    };
-                    logger.info('shard %s successfully downloaded', shard.hash);
-                    shardFinish();
-                  } else {
+                  if (actual !== shard.hash) {
```

Comment:
If not all the bytes are sent, the hash will be incorrect, so this will be triggered for anytime there isn't enough data sent, so the following `} else if (` won't be run.

Reply:
I can upload a 101 byte file and claim it to be 4GB. As long as the hash is correct the farmer will accept it and the bridge will pay the farmer the full 4GB payout.

Reply:
Okay